.azure-pipelines/official-build.yml

```diff
@@ -17,13 +17,13 @@ pr: none
 
 stages:
 - stage: Build
-  pool: sonicbld
+  pool: sonicbld_8c
```

Comment:
What is the difference between sonicbld and sonicbld_8c?

Reply:
sorry ingated unrelated change